playbooks/roles/l2tp-ipsec/templates/streisand-l2tp-iptables.service.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Set iptables rules required to L2TP/IPsec gateway
+After=network.target
+Before=ipsec.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+{% for rule in l2tp_ipsec_firewall_rules %}
+ExecStart=/sbin/{{ rule }}
```

Comment:
A nice follow-up might be trying to make an `ExecStop` that would tear the firewall rules back down. Just mentioning in case you want another task afterwards in a similar area of the codebase :-)

Reply:
Yep, it's definitely a good task to look into afterwards.